### PR TITLE
Replace extension app options with ADK

### PR DESCRIPTION
**Pull request by ystxn**
### Description
- Bump Java projects to use JDK 17
- Remove existing extension app options and files
- Add new extension app options with ADK projects
  - Basic, App View or Message Renderer projects
  - JavaScript or TypeScript
  - Java or Python BDK integration

### CODE REVIEW

File: generators/_lib/util.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+const forge = require('node-forge');
+const axios = require('axios');
+
+const JAVA_BDK_VERSION_DEFAULT = '2.14.1'
```
Comment (yinan-symphony) [inline generators/_lib/util.js:4]: latest version is now 2.14.3

File: generators/_lib/util.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+const PYTHON_BDK_VERSION_DEFAULT = '2.6.1'
+const WDK_VERSION_DEFAULT = '1.6.3'
+const SPRING_VERSION_DEFAULT = '2.7.12'
```
Comment (yinan-symphony) [inline generators/_lib/util.js:7]: since we migrate to jdk17, should we move to spring 3 ? or you think to do so in a next step ?

Comment (ystxn) [reply generators/_lib/util.js:7]: the version check is excluding `RC` and `alpha` tags for now. once we graduate the BDK RC, then spring should default to 3.

https://github.com/finos/generator-symphony/blob/feature/ext-app-overhaul/generators/_lib/util.js#L41

File: generators/_lib/util.js (hunk 1 above)
Comment (ystxn) [reply generators/_lib/util.js]: fixed

**Review by yinan-symphony — APPROVED** [review]
LGTM

**Pull request merged**